src/sudoers/policy.rs

```diff
@@ -94,6 +94,7 @@ impl Policy for Judgement {
 
 pub trait PreJudgementPolicy {
     fn secure_path(&self) -> Option<String>;
+    fn rootpw(&self) -> bool;
```

Comment:
I think at this stage I'd prefer a nicer name than `rootpw` (something slightly more verbose), so someone in the future working in this part doesn't need to consult the `man 5 sudoers` to see what's going on. E.g. `use_root_password`.

Although if we also add `targetpw` as an option, than probably this needs to become a function that provides the user that we need to authenticate as (as an `enum`).

Reply:
With `HARDENED_ENUM_x` values of course :-)

Reply:
Done